src/test/java/emissary/output/roller/JournaledCoalescerTest.java

```diff
@@ -204,13 +202,13 @@ public void testAddFilesWithRoll() throws Exception {
         }
         // verify
         Path bud1Destination = targetBUDPath.resolve(expectedPrefix1);
-        assertThat("Expected target to exist " + bud1Destination.toString(), Files.exists(bud1Destination), equalTo(true));
-        assertThat(bud1Destination.getFileName().toString(), not(equalTo(expectedPrefix2)));
-        assertThat(bud1Destination.getParent(), equalTo(targetBUDPath));
-        assertThat(bud1Destination.getFileName().toString(), equalTo(expectedPrefix1));
-        assertThat(Files.readAllLines(bud1Destination, Charset.defaultCharset()), equalTo(BUD1_LINES));
+        assertTrue(Files.exists(bud1Destination), "Expected target to exist " + bud1Destination);
+        assertNotEquals(bud1Destination.getFileName().toString(), expectedPrefix2);
```

Comment:
Swap positions of unexpected and actual
```suggestion
        assertNotEquals(expectedPrefix2, bud1Destination.getFileName().toString());
```

Reply:
I had to base all of the junit 5 upgrades on #223 -- I'll fix it in #236